proposals/collection-expression-arguments.md

```diff
@@ -526,11 +528,18 @@ IReadOnlyDictionary<string, int> d = [with(comparer: StringComparer.Ordinal), ..
 
 If so, which method signatures are used when binding the arguments?
 
-For `ICollection<T>` and `IList<T>` should we use the accessible constructors from `List<T>`, or specific signatures independent from `List<T>`, say `new()` and `new(int capacity)`?
+For **mutable** interface types, the options are:
+1. Use the accessible constructors from the well-known type required for instantation: `List<T>` or `Dictionary<K, V>`.
+1. Use signatures independent of specific type, for instance using `new()` and `new(int capacity)` for `ICollection<T>` and `IList<T>` (see [*Construction*](#construction) for potential signatures for each interface).
 
-For `IDictionary<TKey, TValue>` should we use the accessible constructors from `Dictionary<TKey, TValue>`, or specific signatures, say `new()`, `new(int capacity)`, `new(IEqualityComparer<K> comparer)`, and `new(int capacity, IEqualityComparer<K> comparer)`?
+For **immutable** interface types, the options are similar:
```

Comment:
```suggestion
For **non-mutable** interface types, the options are similar:
```

Reply:
We use non-mutable in our interface terminology, vs immutable, to avoid confusion with things like the ImmutableXXX apis :)